// lib0/dist/buffer.cjs
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

require('./string-ad04f734.cjs');
require('./environment-ad5e7386.cjs');
var encoding = require('./buffer-884e5a1d.cjs');
require('./map-28a001c9.cjs');
require('./conditions-fb475c70.cjs');
require('./storage.cjs');
require('./function-3695121b.cjs');
require('./array-5347125b.cjs');
require('./set-b596ef38.cjs');
require('./object-fe9c0021.cjs');
require('./binary-ac8e39e2.cjs');
require('./math-08e068f9.cjs');
require('./number-e62129bc.cjs');
require('./error-873c9cbf.cjs');



exports.copyUint8Array = encoding.copyUint8Array;
exports.createUint8ArrayFromArrayBuffer = encoding.createUint8ArrayFromArrayBuffer;
exports.createUint8ArrayFromLen = encoding.createUint8ArrayFromLen;
exports.createUint8ArrayViewFromArrayBuffer = encoding.createUint8ArrayViewFromArrayBuffer;
exports.decodeAny = encoding.decodeAny;
exports.encodeAny = encoding.encodeAny;
exports.fromBase64 = encoding.fromBase64;
exports.toBase64 = encoding.toBase64;
